doc: update known issues for NCS v2.6.0 (#14185)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

Amazon Sidewalk
===============

> 

Reviewer: ```suggestion

.. rst-class:: v2-6-0

KRKNWK-11630: BUS fault on FSK during the FACTORY_RESET call.
  Bluetooth LE link is deinitialized, however, a race condition might occur where an event added by the Bletooth LE link is not removed. 
  Acting on the event without a valid Bluetooth LE handle leads to dereference of ``NULL``.
  To avoid the error, you need to check for ``NULL`` before the event is processed in the event queue.
  The issue requires a fix in the Sidewalk stack.
 
  **Workaround:** Perform a factory reset as the error is recoverable.
 
.. rst-class:: v2-6-0

KRKNWK-11630: Advertising fails to start (``err -12``) upon registration.
  The Sidewalk end device is trying to restart advertising, for example, on time sync lost. 
  This happens even when the device is connected through Bluetooth LE, leading to an error.
  The Bluetooth LE connection management requires a fix in the Sidewalk stack.
 
  **Workaround:** The issue fixes upon automatic restart of the Bluetooth LE advertising.
 
.. rst-class:: v2-6-0
 
KRKNWK-17653: Assertion fails during the stress test (pressing different buttons).
  There is a corner case in dynamic heap allocation which leads to a double free error.

  **Workaround:**  The device resets after assertion and functions normally.

```

Author: Done

  The application builds successfully, but the error might obfuscate other important warnings.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

KRKNWK-17374: Sporadic Zephyr fatal error after disconnecting on FSK
  After disconnecting on FSK, Zephyr fatal error occurs due to assertion in the semaphore module.
  The error reproduces rarely (once per a few days).
  The device resets automatically in the release mode, however, in the debug mode it needs to be reset manually.
  Currently, this issue occurs for Sidewalk v1.14 libraries, and it will be fixed in a future version.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

KRKNWK-17035: Sensor monitor uplink messages are lost when the notification period is longer than 30 seconds
  If the notification period is set to longer than 30 seconds, sensor monitor uplink messages are lost.

  **Workaround:** The notification period is set to 15 seconds by default.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

KRKNWK-14583: Bus fault after flash, before the :file:`Nordic_MFG.hex` data flash
  For sub-GHz samples, when the :file:`Nordic_MFG.hex` file is missing, the device throws a hard fault during initializing the Sidewalk stack.

>   Proper error handling will be implemented, but the temporary solution is to write a manufacturing hex to the device and reset it.

Reviewer: The following issues are fixed in 2.6.0:
KRKNWK-17860
KRKNWK-17800
KRKNWK-17244
KRKNWK-17374
KRKNWK-17035
KRKNWK-14583

Amazon Sidewalk
===============

.. rst-class:: v2-6-0

KRKNWK-18465: BUS fault on FSK during the FACTORY_RESET call
  The Bluetooth LE link is deinitialized.

>   However, a race condition might occur where an event added by the Bletooth LE link is not removed.

Reviewer 2: ```suggestion
  However, a race condition might occur where an event added by the Bluetooth® LE link is not removed.
```

Reviewer 3: Also, could remove the trademark from line no: 87 and make the above as "*Bluetooth®*"

  The Bluetooth LE link is deinitialized.
  However, a race condition might occur where an event added by the Bletooth LE link is not removed.
  Acting on the event without a valid Bluetooth LE handle leads to dereference of ``NULL``.
  To avoid the error, you need to check for ``NULL`` before the event is processed in the event queue.
  The issue requires a fix in the Sidewalk stack.

  **Workaround:** Perform a factory reset as the error is recoverable.

.. rst-class:: v2-6-0

KRKNWK-18511: Advertising fails to start (``err -12``) upon registration
  The Sidewalk end device is trying to restart advertising, for example, on time sync lost.
  This happens even when the device is connected through Bluetooth LE, leading to an error.
  The Bluetooth LE connection management requires a fix in the Sidewalk stack.

  **Workaround:** The issue fixes upon automatic restart of the Bluetooth LE advertising.

.. rst-class:: v2-6-0

KRKNWK-18546: High current consumption for the Sidewalk application
  In the idle state, the release version of the Sidewalk samples consumes too much current (~240 µA) due to enabled UART in the network core.

>   **Affected platforms:** nRF5340 DK

Author: Removed.

> KRKNWK-18499: Assertion fails during the stress test (pressing different buttons)
>   There is a corner case in the dynamic heap allocation, which leads to a double free error.
> 
>   **Workaround:**  The device resets after assertion and functions normally.

Reviewer: ```suggestion
KRKNWK-18546: High current consumption for the Sidewalk application
  In the idle state, the release version of the Sidewalk samples consumes too much current (~240µA) due to enabled UART in the network core.

  **Affected platforms:** nRF5340 DK
```
This entry didn't stay here for too long - KRKNWK-18499 is fixed already:) adding a new one instead

Author: Thanks Anna. Any workaround?

Reviewer: None for now

The issues in this section are related to the :ref:`ug_bt_mesh` protocol.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

Reviewer 4: This has been resolved for v2.6.0

NCSDK-21625: Advertisements of Bluetooth Mesh GATT services are not stopped by :c:func:`bt_mesh_suspend` and not resumed by :c:func:`bt_mesh_resume`
  Functions :c:func:`bt_mesh_suspend` and :c:func:`bt_mesh_resume` do not work together with functions :c:func:`bt_disable` and :c:func:`bt_enable`.

  **Workaround:** To disable node identity advertisement, use ``bt_mesh_subnet_node_id_set`` instead.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

Reviewer 4: This has been resolved for v2.6.0

NCSDK-23087: Unsolicited Time Status messages rewrite periodic publishing TTL to zero forever
  The Time models specification mandates publishing unsolicited Time Status messages with TTL field value set to ``0``.
  The implementation rewrites the TTL field to ``0``, but does not write the initial value back, resulting in losing the initial value.

  **Workaround:** Configure the initial TTL value after an unsolicited Time Status message is sent.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

Reviewer 4: This has been resolved for v2.6.0

NCSDK-23220: The Heartbeat Publication Status message may be malformed after provisioning
  After provisioning and obtaining the Composition Data, reading the Heartbeat Publication and the Heartbeat Publication Status will contain garbage in the NetKeyIndex field.
  The reason for this is that the field was not initially cleared.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

Reviewer 4: This has been resolved for v2.6.0

  When running the :ref:`bluetooth_central_dfu_smp` sample, the :kconfig:option:`CONFIG_BT_SMP` configuration must be aligned between this sample and the Zephyr counterpart (:zephyr:code-sample:`smp-svr`).
  However, security is not enabled by default in the Zephyr sample.

.. rst-class:: v2-5-2

DRGN-21390: The `disconnected` callback might not get called on nRF5340
  The Bluetooth host running on the nRF5340 application core might deadlock on disconnection.

>   This is due to a recent bugfix in the Softdevice Controller triggering a bug in the ATT queuing layer.

Reviewer 2: ```suggestion
  This is due to a recent bugfix in the SoftDevice Controller triggering a bug in the ATT queuing layer.
```

  **Workaround:** Do not enable the :kconfig:option:`CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE` option.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1

Reviewer 4: This has been resolved for v2.6.0

NCSDK-21780: Sensor types with floating point representation lose precision
  Sensor types with floating point representation lose precision when converted to ``sensor_value`` in the sensor API callbacks.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1

Reviewer 4: Add wontfix

Author: Done

> The issues in this section are related to the :ref:`ug_matter` protocol.

Reviewer 5: Please also add (it should be the scope of Matter samples I suppose):

KRKNWK-18242: Thermostat sample does support the AUTO system mode
  AUTO system mode is supported in the thermostat device but it is not reflected in the
  in the data model and the feature map indicates that AUTO system mode is not supported.
 
  **Workaround:** Set the feature map to `0x23` in the `thermostat.zap` file and regenerate data model, so that it  indicates that the AUTO mode is supported by thermostat device.

Author: Just for v2-6-0?

Reviewer 5: Sorry, starting from 2.5.0 release:

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``836390ed636ca36126dbcbe763d0f127626cba8d``).

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

Reviewer 2: typo Verfier in line 386

> 

Reviewer 6: Please add:

.. rst-class:: v2-5-0

KRKNWK-18572: Bus fault when resetting the Zigbee light switch sample
  Usage of :kconfig:option:`CONFIG_RAM_POWER_DOWN_LIBRARY` leads to a bus fault at reset.

  **Workaround:** Choose  :kconfig:option:`CONFIG_MINIMAL_LIBC` as  :kconfig:option:`CONFIG_LIBC_IMPLEMENTATION`.

Author: I believe this would go under Zigbee samples then.

Reviewer 6: Well, it actually affects any usage of `CONFIG_RAM_POWER_DOWN_LIBRARY`.

Author: So back under Zigbee protocol?

The issues in this section are related to the :ref:`serial_lte_modem` application.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0

Reviewer 7: ```suggestion
.. rst-class:: wontfix v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0
```

  **Workaround:** Set the :ref:`CONFIG_SLM_UART_RX_BUF_SIZE <CONFIG_SLM_UART_RX_BUF_SIZE>` Kconfig option to ``2048`` to ensure that there is adequate buffer space to receive traffic from the Connectivity Bridge and disable the hw-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.
  If even larger buffer spaces are required, the :kconfig:option:`CONFIG_BRIDGE_BUF_SIZE` Kconfig option for Connectivity Bridge and the :ref:`CONFIG_SLM_UART_RX_BUF_SIZE <CONFIG_SLM_UART_RX_BUF_SIZE>` Kconfig option for SLM, must be set accordingly.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

Reviewer 7: ```suggestion
.. rst-class:: v2-5-0
```

  For example, if a stream is in the playing state but media control is in the pause state, it might not be possible to play or pause.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0

OCT-2725: Data can be overwritten when publishing configurations too fast on :ref:`zephyr:zbus`
  When, for example, setting up a bidirectional stream, the configurations for sink and source might be received too fast, resulting in one of the configurations being lost.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCT-2765: BIS headset stack overflow if gateway is periodically reset
  A BIS headset might get stack overflow if the BIS source is reset several times with short intervals.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCT-2766: BIS headset might get into an infinite loop
  If a BIS headset is paused, the BIS broadcaster is reset, and if the headset is unpaused, it will end up in an infinite loop.

  **Affected platforms:** nRF5340 Audio DK

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

> 

Reviewer 8: Please add:

OCT-2172:
The headset volume is not stored persistently. This means the volume will fall back to default level after a reset.

Author: Only for v2-6-0?

Reviewer 8: It has been for earlier releases as well, but ATM we don't know when this issue was introduced.

  The TX power for a connection cannot be less than -20 dBm.
  Controller still output -20 dBm if the setting is -40 dBm.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-208: There is a chance that the scan report only shows legacy-ADV without ADV-EXT
  This might happen if a broadcast sink is moved in and out of radio range from the broadcast source or if the broadcast source is being rapidly reset.
  It will cause a broadcast sink to not be able to finish PA sync.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-217: The controller can get unresponsive when a broadcast sink tries to sync to some broadcast sources
  This is only an issue if a broadcast sink tries to sync to certain broadcast sources made by other vendors.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-218: The BIG info is not sent from the controller when a broadcast sink tries to sync to some broadcast sources
  This is only an issue if a broadcast sink tries to sync to certain broadcast sources made by other vendors.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-223: The controller asserts on the CIS gateway if two paused headsets reset

>   If a CIS headset pair is reset simultaneously a few times while remaining paused, the controller on the gateway might get unresponsive.

Reviewer 8: Please add:
OCX-NONE: OCT-2897:
Use of interleaved packing on the controller may cause left headset to disconnect.

Author: Only for v2-6-0?

Reviewer 8: Yes

  The TX power for a connection cannot be less than -20 dBm.
  Controller still output -20 dBm if the setting is -40 dBm.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-208: There is a chance that the scan report only shows legacy-ADV without ADV-EXT
  This might happen if a broadcast sink is moved in and out of radio range from the broadcast source or if the broadcast source is being rapidly reset.
  It will cause a broadcast sink to not be able to finish PA sync.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-217: The controller can get unresponsive when a broadcast sink tries to sync to some broadcast sources
  This is only an issue if a broadcast sink tries to sync to certain broadcast sources made by other vendors.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-218: The BIG info is not sent from the controller when a broadcast sink tries to sync to some broadcast sources
  This is only an issue if a broadcast sink tries to sync to certain broadcast sources made by other vendors.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

OCX-223: The controller asserts on the CIS gateway if two paused headsets reset

>   If a CIS headset pair is reset simultaneously a few times while remaining paused, the controller on the gateway might get unresponsive.

Author: Is this only for v2-6-0?

>   If a CIS headset pair is reset simultaneously a few times while remaining paused, the controller on the gateway might get unresponsive.

Reviewer 8: It may be for earlier releases as well, but ATM we don't know when this issue was introduced.

> 

Reviewer 9: Please add a new issue (no workaround available yet):
```
NCSDK-26424: Directed advertising in :ref:`peripheral_hids_mouse` does not start after disconnecting from a bonded peer.
When the sample disconnects from a peer, after successful pairing and subscription to reports, it cannot re-connect because directed advertising does not start.
```

Author: Done

Bluetooth Mesh samples
======================

.. rst-class:: v2-6-0

NCSDK-26388: Compilation of Mesh Light sample can create an image without MCUboot
  This can happen when compiled with the point-to-point DFU overlay and ``--sysbuild`` option.

  **Workaround:** To get a correct image with MCUboot, do not use the ``--sysbuild`` option.

.. rst-class:: v2-6-0

> NCSDK-26403: Point-to-point DFU procedure Test and Confirm with erasing application area cannot succeed in the Device Manager app on iOS for the Mesh Light sample

Reviewer 4: ```suggestion
NCSDK-26403: Point-to-point DFU procedure 'Test and Confirm' with erasing application area cannot succeed in the Device Manager app on IOs for Mesh Light sample. After image uploading and resetting the device the mobile application cannot receive confirmation back and the whole procedure fails.
```

Author: Done.

  **Affected platforms:** nRF9161

> 

Reviewer 10: Please insert these two new issues:

```rst
.. rst-class:: v2-6-0

IRIS-8456: Wi-Fi builds of the :ref:`nrf_cloud_multi_service` crash and reboot if no Wi-Fi APs are visible for more than a few minutes.

.. rst-class:: v2-6-0
  
IRIS-8465: CoAP builds of the :ref:`nrf_cloud_multi_service` stall if connectivity is lost.
  If PDN detaches for too long while the sample is connected, the sample is unable to reconnect to nRF Cloud after PDN returns.  
```

```

The issues in this section are related to :ref:`app_bootloaders`.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

Reviewer 11: Issue has been fixed with PR https://github.com/nrfconnect/sdk-nrf/pull/12846.
The NCSDK-24203 is currently marked as Closed.

The issues in this section are related to :ref:`app_build_system`.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

NCSIDB-816: nanopb/protoc not specified correctly by the |NCS| Toolchain

>   The |NCS| Toolchain includes the nanopb/protoc tool when installed, but the path to the tool is not propagated correctly to the |NCS| build system.

Reviewer 12: This has been fixed since v2.3

The issues in this section are related to :ref:`app_dfu`.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0

NCSDK-21790: Errors during DFU when using nRF Connect for mobile app
  MCUmgr is incorrectly reporting an error when DFU is performed using the nRF Connect for mobile app.
  The DFU functionality is working as expected.

  **Affected platforms:** nRF5340

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0

NCSDK-21379: Single slot network core updates on nRF5340 does not work properly

>   Currently, a bug in the MCUboot code causes network core updates to not be applied when using the nRF5340 in a single slot configuration.

Reviewer 11: Issue has been fixed https://github.com/nrfconnect/sdk-mcuboot/pull/257.
The NCSDK-21379 is currently marked as resolved.

The issues in this section are related to :ref:`nrfxlib:crypto`.

.. rst-class:: v2-6-0

NSCDK-26412: Updating to TF-M 2.0 using Mbed TLS 3.5.2 introduced a regression in resolving legacy crypto configurations from ``PSA_WANT_ALG_XXXX`` configurations
  Wi-Fi samples enabling OpenThread are affected by this bug as well as possible use cases with a dependency on some legacy features while using PSA crypto APIs.
  Unfortunately, the issue is not caught at build-time.
  It only appears at runtime.

>   **Workaround:** Enable TF-M on TrustZone-enabled devices to circumvent this issue, or manually add missing legacy configurations in the :file:`psa_crypto-config.h.template` and :file`psa_crypto_want_config.h.template` files.

Reviewer 2: ```suggestion
  **Workaround:** Enable TF-M on TrustZone-enabled devices to circumvent this issue, or manually add missing legacy configurations in the :file:`psa_crypto-config.h.template` and :file:`psa_crypto_want_config.h.template` files.
```

The issues in this section are related to :ref:`nrfxlib:nrf_802154`.
In addition to the known issues listed here, see also :ref:`802.15.4 Radio driver limitations <nrf_802154_limitations>` for permanent limitations.

> 

Reviewer 13: For section `802.15.4 Radio driver` please add:

```
.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0

KRKNWK-18589: Timestamps for delayed operations triggering very shortly after a sleep request may be very inaccurate.
```

  When the CIG or BIG is configured with an ``ISO_Interval`` that equals the ``SDU_Interval``, there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference.
  If several SDUs are transmitted during each ``ISO_Interval``, meaning that it is larger than the ``SDU_Interval``, our implementation of the LE Read ISO TX Sync command returns a unique SDU synchronization reference for each SDU.

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0

DRGN-20444: The HCI LE Create Connection command and the HCI LE Extended Create Connection command overwrite scan parameters
  This happens when the HCI LE Create Connection command or the HCI LE Extended Create Connection command is called after the scan parameters are set.

  **Workaround:** Set the scan parameters with the HCI LE Set Scan Parameters command or the HCI LE Set Extended Scan Parameters command again.

> .. rst-class:: v2-5-1 v2-5-0 v2-4-2 v2-4-1 v2-4-0

Reviewer 14: The below needs to be added:
.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
DRGN-20762: LE Set Periodic Advertising Subevent Data could fail unexpectedly if interrupted
The LE Set Periodic Advertising Subevent Data command could fail when providing data at the same time as an AUX_SYNC_SUBEVENT_IND was sent.

.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 … v1-8-0
DRGN-20815: A packet might not be received when sent at the instant of a Channel Map Update
This could happen when the controller is acting as slave, while the master is sending a Channel Map Update.

.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0
DRGN-20432: LE Set Periodic Advertising Response Data command can assert if host behaves incorrectly
This could happen if the LE Set Periodic Advertising Response Data command was issued more than once without fetching the Command Complete Event. 

.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0….
DRGN-20832: The controller would assert during cooperative active scanning or when running a cooperative initiator
This could happen when the controller was about to send a scan request or connect indication. 

.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0….v1.5.0…
DRGN-20862: The nRF5340 DK consumed too much current while scanning
This could happen if the controller was running with TX power higher than 0 dB.

.. rst-class:: v2-5-1 v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0….v1.5.0…
DRGN-21085: The controller would stop sending ACL data packets to the host when controller to host flow control was enabled
This could happen when a disconnection occurred before the host had issued the Host Number of Complete Packets command for the remaining ACL data packets.

.. rst-class:: v2-5-1 v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0….v1.5.0…
DRGN-21253: Rare assert on the scanner
The scanner may assert when it schedules the reception of the next advertising packet.

Author: Done

The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.

> 

Author: Done

DRGN-18568: Using :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option lowers the value of radio output power
  The actual value is lower than the default one in case the :kconfig:option:`CONFIG_BT_CTLR_TX_PWR_ANTENNA` or :kconfig:option:`CONFIG_BT_CTLR_TX_PWR` Kconfig options are used together with the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

Reviewer 14: rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

  **Workaround:** Enable :kconfig:option:`CONFIG_MCUBOOT_CLEANUP_ARM_CORE`` in MCUboot configuration.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0

Reviewer 15: ```suggestion
.. rst-class:: v2-5-2 v2-5-1 v2-5-0
```

Fixed in 2.6.0

LwM2M engine blocks all composite operations
  Due to a bug in LwM2M engine, all composite operations get a return code of ``4.01 - Unauthorized``.
  This has been reported in `Zephyr issue #6412 <https://github.com/zephyrproject-rtos/zephyr/issues/64012>`_.

  **Workaround:** To fix the error, cherry-pick commits from the upstream `Zephyr PR #64016 <https://github.com/zephyrproject-rtos/zephyr/pull/64016>`_.

> .. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0

Reviewer 15: ```suggestion
.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0
```

Fixed in 2.6.0
